.NET/Microsoft.Recognizers.Text.DateTime/Extractors/BaseDateTimeExtractor.cs

```diff
@@ -222,7 +222,14 @@ public List<Token> TimeOfTodayAfter(string text, DateObject reference)
 
             var matches = this.config.SimpleTimeOfTodayAfterRegex.Matches(text);
             foreach (Match match in matches)
-            {
+            {

```

Comment:
This is not exactly what I meant. I meant modifying the regexes like BaseDateTime.HourRegex in the BaseDateTime.YAML file, if possible.
I worry that this new implementation might cause a perf hit. Can you measure the performance impact of it vs changing the YAML file?
Also, could you add some spec test cases in non-English languages to show that the fix works in them too?
Thanks.

Reply:
ah I see, ok I will do and will check if it is possible modifying BaseDateTime.YAML.

Reply:
moved solution to Base-DateTime.yaml (but kept themodifications to base code in Java because of the problems with negative lookbehinds).

Reply:
noticed also that FilterAmbiguity is not implemented in JavaScript MergedExtractor (this causes problems for example with the input "Mi temperatura era 37.1 por la mañana" in Spanish where "Mi" is extracted as weekday (abbreviation of "miercoles")). 
Should I open an issue?

Reply:
Please open an issue for the ambiguity filter in JS and mention this spec case there. Thanks.

Reply:
And this check should now be removed from .NET, right?

Reply:
yes sorry, I forgot to update the .NET files.